de_workloads/shared_resources/data_factory/adf_linked_services.tf

```diff
@@ -31,15 +31,15 @@ resource "azurerm_data_factory_linked_service_azure_databricks" "ls_Databricks_S
 
   new_cluster_config {
     node_type             = "Standard_DS3_v2"
-    cluster_version       = "13.0.x-scala2.12"
+    cluster_version       = "13.3.x-scala2.12"
```

Comment:
Updated to 13.3 which is LTS - https://docs.databricks.com/en/release-notes/runtime/index.html